korge/src/commonMain/kotlin/com/soywiz/korge/ui/UIComboBox.kt

```diff
@@ -0,0 +1,142 @@
+package com.soywiz.korge.ui
+
+import com.soywiz.korge.html.*
+import com.soywiz.korge.input.*
+import com.soywiz.korge.view.*
+import com.soywiz.korim.color.*
+import com.soywiz.korio.async.*
+
+inline fun <T> Container.uiComboBox(
+	width: Number = 192.0,
+	height: Number = 32.0,
+	selectedIndex: Int = 0,
+	items: List<T>,
+	verticalScroll: Boolean = true,
+	textFont: Html.FontFace = defaultUIFont,
+	skin: UISkin = defaultUISkin,
+	showSkin: UISkin = skin,
+	hideSkin: UISkin? = null,
+	upSkin: UISkin? = null,
+	downSkin: UISkin? = null,
+	scrollbarSkin: UISkin = skin,
```

Comment:
Why not include the colors/images used for the UI as part of a single UISKin?

Reply:
We can create different skins for different views, so each view skin will have a unique set of parameters to specify its own state, it's a quite good solution. But what about the case when a developer want each component of a view (for example, each button of uiComboBox) to look different? In this case, I suppose, we need different skins for each component.